src/search/landmarks/exploration.h

```diff
@@ -50,12 +42,12 @@ struct ExUnaryOperator {
     int base_cost; // 0 for axioms, 1 for regular operators
```

Comment:
I think this comment has been wrong ever since we started supporting action costs. Or is it really always 1 for regular operators? (But I suppose that's unrelated to this issue.)

Reply:
Are you fine with just removing it completely?

Reply:
Yes, as it doesn't matter for testing relaxed reachability. But if we do more changes like these in the same issue, I'd suggest rerunning the experiments because it is always good to have experimental results for the things that we merge. If running another experiment is too onerous, that may be sufficient reason to leave it to the next issue.

Reply:
I see. Sure, just remove the comment. Regarding when we need experiments: in this case we do the experiments to a large extent to test the performance impact, not just to test the correctness of the modification. For this, many of the individual changes had an impact.